src/TransitionSystems/quotient.rs

```diff
@@ -452,6 +453,21 @@ impl TransitionSystem for Quotient {
     fn get_dim(&self) -> ClockIndex {
         self.dim
     }
+
+    fn get_combined_decls(&self) -> Declarations {
+        let mut clocks = HashMap::new();
+        let mut ints = HashMap::new();
+        for decl in [
+            self.S.get_combined_decls(),
+            self.T.get_combined_decls(),
+            self.decls.clone(),
+        ] {
+            clocks.extend(decl.clocks);
+            ints.extend(decl.ints)
+        }
+
+        Declarations { ints, clocks }
+    }
```

Comment:
Doesn't this do the same as the default implementation?

Reply:
Quotient has also a declaration, but I have changed the default implementation get_combined_decls, so this code is also removed :) @t-lohse